content/github/working-with-github-pages/creating-a-github-pages-site-with-jekyll.md

```diff
@@ -63,24 +63,30 @@ Before you can use Jekyll to create a {% data variables.product.prodname_pages %
  $ git checkout --orphan gh-pages
  # Creates a new branch, with no history or contents, called gh-pages and switches to the gh-pages branch
  ```
- 7. To create a new Jekyll site, use the `jekyll new` command, replacing _VERSION_ with the current dependency version for Jekyll. For more information, see "[Dependency versions](https://pages.github.com/versions/)" on the {% data variables.product.prodname_pages %} site.
-    - If you installed Bundler:
-      ```shell
-      $ bundle exec jekyll <em>VERSION</em> new .
-      # Creates a Jekyll site in the current directory
-      ```
-    - If you don't have Bundler installed:
-     ```shell
-     $ jekyll <em>VERSION</em> new .
-     # Creates a Jekyll site in the current directory
-     ```
-8. Open the Gemfile that was created and follow the instructions in the Gemfile's comments to use {% data variables.product.prodname_pages %}.
-  ![Instructions for updating Gemfile](/assets/images/help/pages/gemfile-instructions.png)
-9. Update the `gem "github-pages"` line so that the line looks like this, replacing _VERSION_ with the current dependency version for `github-pages`. For more information, see "[Dependency versions](https://pages.github.com/versions/)" on the {% data variables.product.prodname_pages %} site.
-```shell
-gem "github-pages", "~> <em>VERSION</em>", group: :jekyll_plugins
-```
+7. To create a new Jekyll site, use the `jekyll new` command:
+   ```shell
+   $ jekyll new .
+   # Creates a Jekyll site in the current directory
+   ```
+8. Open the file called `Gemfile` that was created, and find the line that looks like this:
+   ```shell
+   # gem "github-pages", group: :jekyll_plugins
+   ```
+   and update it so that it looks like this:
+   ```shell
+   gem "github-pages", "~> 209", group: :jekyll_plugins
+   ```
+   If there is a newer version of the `github-pages` gem, replace `209` above with the latest version that is listed in [Rubygems](https://rubygems.org/gems/github-pages).
+   Then find the line that looks like this:
+   ```shell
+   gem "jekyll", "~> 4.2.0"
+   ```
+   and disable it by adding a `#` to it:
+   ```shell
+   # gem "jekyll", "~> 4.2.0"
+   ```
```

Comment:
I don't think it is good practice to include explicit version numbers which likely will not match what the user sees.
~~~suggestion
   Comment out the Gemfile line which starts with:
  ```shell
   gem "jekyll"
   ```
  The correct version Jekyll will be installed as a dependency of the `github-pages` gem.
~~~
I think instructing users on how to comment out lines in a Gemfile is out of scope for this guide so this should be sufficient.